rep-2005.rst

```diff
@@ -500,6 +500,31 @@ All items on the list for the next distro should already be released into the ro
   * `urdf/urdf <https://index.ros.org/p/urdf/>`_
   * `urdfdom/urdfdom <https://index.ros.org/p/urdfdom/>`_
 
+* Web
+
+  * `foxglove/ros1 <https://github.com/foxglove/ros1>`_
```

Comment:
Thanks! As discussed below, we have removed the github.com/foxglove packages for now, and the remaining github.com/RobotWebTools packages (other than those already linked to index.ros.org) do not have index.ros.org entries because they are NPM packages, and to the best of my knowledge index.ros.org does not yet allow tracking of javascript/npm packages.